fem/fe.cpp

```diff
@@ -7950,6 +7950,22 @@ VectorTensorFiniteElement::VectorTensorFiniteElement(const int dims,
      cbasis1d(poly1d.GetBasis(p, VerifyClosed(cbtype))),
      obasis1d(poly1d.GetBasis(p - 1, VerifyOpen(obtype))) { }
 
+VectorTensorFiniteElement::VectorTensorFiniteElement(const int dims,
+                                                     const int d,
+                                                     const int p,
+                                                     const int obtype,
+                                                     const int M,
+                                                     const DofMapType dmtype)
+   : VectorFiniteElement(dims, GetTensorProductGeometry(dims), d,
+                         p, M, FunctionSpace::Pk),
+     TensorBasisElement(dims, p, obtype, dmtype),
+     cbasis1d(poly1d.GetBasis(p, VerifyOpen(obtype))),
+     obasis1d(poly1d.GetBasis(p, VerifyOpen(obtype)))
```

Comment:
There are now 2 constructors for `VectorTensorFiniteElement`, and their behavior is different (e.g. `p` here, `p-1` in the other). Does this mean the other constructor should be not allowed in the 1D case?